modules/microshift-lvms-system-requirements.adoc

```diff
@@ -11,7 +11,7 @@ Using LVMS in {product-title} requires the following system specifications.
 [id="lvms-volume-group-name_{context}"]
 == Volume Group Name
 
-The default integration of LVMS assumes a volume group named `rhel`. Prior to launching, the `lvmd.yaml` configuration file must specify an existing volume group on the node with sufficient capacity for workload storage. If the volume group does not exist, the node controller will fail to start and enter a `CrashLoopBackoff` state.
+The default integration of LVMS assumes a volume group named `rhel`. Prior to launching, the `lvmd.yaml` configuration file must specify an existing volume group on the node with sufficient capacity for workload storage. If the volume group does not exist, the node controller will fail to start and enter a `CrashLoopBackoff` state. You can change the default name of the volume group in the configuration file.
```

Comment:
Would benefit from a doc link to modules/microshift-lvms-configuring.adoc.

Reply:
I wish we could link inside modules, but we can't. I did add a reference to the section.